antarest/study/storage/variantstudy/model/command/create_cluster.py

```diff
@@ -103,8 +103,7 @@ def _validate_against_version(self) -> Self:
         validate_thermal_cluster_against_version(self.study_version, self.parameters)
         return self
 
-    @override
-    def _apply_config(self, study_data: FileStudyTreeConfig) -> Tuple[CommandOutput, Dict[str, Any]]:
+    def validate_data(self, study_data: FileStudyTreeConfig) -> Tuple[CommandOutput, Dict[str, Any]]:
```

Comment:
naming: it's maybe a little misleading to name this way, because actually it does not only validate the data (which looks like a read only operation), it also updates the "config".

Also, we can now do simpler than return a dict, since we are not constrained anymore by the parent class signature: we could for example just return the cluster ID.
The successful `CommandOutput` can be created at the end of the `_apply` method instead of here, it seems a little weird now to create it here.